Chore: Minor refactorings (#8)
charmed-osm/srs-enb-ue-operator: README.md

Open-source 4G EnodeB and User emulators developed by [Software Radio Systems (SRS)](https://www.srslte.com/).

> ## Installation and configuration

Reviewer: I think the `usage` wording is pretty universal

Author: Reverted.

Open-source 4G EnodeB and User emulators developed by [Software Radio Systems (SRS)](https://www.srslte.com/).

## Installation and configuration

> ```bash

Reviewer: You can remove those 3 lines

Author: Which 3 lines?

Reviewer: The lines starting where I added the comment

Author: Already removed.

juju deploy charmed-osm-srs-enb-ue --channel edge
```

> The easiest method is to configure and deploy at the same time. For it create a YAML configuration file:

Reviewer: ```suggestion
Create a YAML configuration file with the necessary charm config:
```

Author: Fixed.

```yaml
---
srs-enb-ue:
  bind-address-subnet: <ipv4 address>
  enb-name: <enb name>
  enb-mcc: <mcc>
  enb-mnc: <mnc default>
  enb-rf-device-name: <zmq>
  enb-rf-device-args: <RF Device Name.>
  ue-usim-algo: <The authentication algorithm to use (MILENAGE or XOR).>
  ue-nas-apn: <NAS Access Point Name (APN).>
  ue-device-name: <UE Device Name.>
  ue-device-args: <UE Device arguments.>
```

And run the deploy command:

```bash
juju deploy charmed-osm-srs-enb-ue --config <yaml config file>
```

> 

Reviewer: All those lines are not necessary in my opinion (33-47). Here we don't want to tell the user how to use charms/Juju.
